README.md

```diff
@@ -1,70 +1,155 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+<div align="center">
+  
+  <br/>
 
-In the project directory, you can run:
+# React Capstone TV Shows Metrics
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+</div>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+<!-- TABLE OF CONTENTS -->
 
-### `npm test`
+# 📗 Table of Contents
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+  - [🚀 Live Link](#live-link)
+  -[🙃 video explanation](#video-explanation)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-### `npm run build`
+<!-- PROJECT DESCRIPTION -->
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+# Live app<a name="live-link"></a>
+** https://paulshaun-kdtk.github.io/React_Capstone_Metrics/
 
```

Comment:
- [ ] Good job by adding the live demo link but as mentioned by the previous reviewer you should provide a functional link, a link that works correctly. When I open the live demo link I see the README file instead of the output of your project. Kindly fix that 🙏🏾 🙃 

https://github.com/paulshaun-kdtk/React_Capstone_Metrics/assets/74501523/e30fbb61-ce26-4de0-95c1-1796f77aba37

Reply:
it's a react application with a server that needs actual hosting ... I'm not sure we can do that on github... and if we can please kindly share how

otherwise I'll factor to remove the live link from my README.md file

Reply:
You can use Render, Netlify,...  You can use this [link](https://create-react-app.dev/docs/deployment/) for documentation

Reply:
thank you